test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class PostsControllerTest < ActionController::TestCase
+  setup do
+    @post = posts(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:posts)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create post" do
+    assert_difference('Post.count') do
+      post :create, post: { description: @post.description, image: @post.image, title: @post.title }
+    end
+
+    assert_redirected_to post_path(assigns(:post))
+  end
+
+  test "should show post" do
+    get :show, id: @post
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @post
+    assert_response :success
+  end
+
+  test "should update post" do
+    patch :update, id: @post, post: { description: @post.description, image: @post.image, title: @post.title }
```

Comment:
Metrics/LineLength: Line is too long. [110/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)